README.md

```diff
@@ -98,6 +98,12 @@ Fore more information please refer to the [Git documentation, Formatting and Whi
 
 ----
 
+## Running Spring Cloud Data Server
+
+For executing the SCDF server, the main entry point is [SCDF Server Folder](./spring-cloud-dataflow-server). It has all the docker-compose defined to launch the server locally. One can also launch it via IDE if they can select the `src` folder of [SCDF Server Folder](./spring-cloud-dataflow-server/src) and enable to debug the main server file: [DataFlowServerApplication](./spring-cloud-dataflow-server/src/main/java/org/springframework/cloud/dataflow/server/single/DataFlowServerApplication.java)
```

Comment:
Not sure what this is suppose to mean? 
* For debugging streaming apps with the docker-compose installation follow the: https://dataflow.spring.io/docs/installation/local/docker-customize/#debug-stream-applications instructions.
* For debugging SCDF server with the docker-compose installation follow the: https://dataflow.spring.io/docs/installation/local/docker-customize/#debug-data-flow-server instructions.
* For debugging Skipper server with the docker-compose installation follow the: https://dataflow.spring.io/docs/installation/local/docker-customize/#debug-skipper-server instructions.

Reply:
Thanks for this comment. 

The purpose of this update was to provide first-hand information from the main README file to proceed to the DataFlow server folder for starting the application. 

When I read the documentation for the first to understand how to start, the README file was not having a section on how to start the application.

Anyhow, I now see that the same is maintained in another doc.